boards/unphone7.json

```diff
@@ -0,0 +1,40 @@
+{
+  "build": {
+    "arduino":{
+      "ldscript": "esp32_out.ld"
+    },
+    "core": "esp32",
+    "extra_flags": [
+      "-DUNPHONE_SPIN=7",
+      "-DARDUINO_FEATHER_ESP32"
+    ],
+    "f_cpu": "240000000L",
+    "f_flash": "40000000L",
+    "flash_mode": "dio",
+    "mcu": "esp32",
+    "variant": "feather_esp32"
```

Comment:
Are you sure the `feather_esp32` variant is suitable for your board?
Same question for `-DARDUINO_FEATHER_ESP32`.

Reply:
In the early versions we duplicated the Huzzah32 Feather pinouts and users selected that board definition in platformio.ini

So this does work for versions before 8, when we move over to the ESP32S3 and pin mappings become a little different.

We've tested all of these platformio board defs with various test rigs, including: https://gitlab.com/hamishcunningham/unphone/-/blob/master/examples/2022/platformio.ini